c3/optimizers/optimalcontrol_robust.py

```diff
@@ -1,12 +1,12 @@
 import time
 import hjson
 import tensorflow as tf
-from c3.optimizers.c1 import C1
+from c3.optimizers.optimalcontrol import OptimalControl
 from c3.utils.utils import jsonify_list
 import numpy as np
 
 
-class C1_robust(C1):
+class OptimalControlRobust(OptimalControl):
```

Comment:
Seems like the more commonly used term is Robust Optimal Control. I know it probably doesn't make much of a difference but it might be sensible to see if we want to have the more typical names. ([arXiv:2103.15716](https://arxiv.org/abs/2103.15716))